auction-app-frontend/src/components/BidsTab/BidsTab.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState, useEffect } from "react";
+
+import { ProductBidsTable, LoadingComponent, ErrorComponent } from "src/components";
+
+import { useUser } from "src/store/UserContext";
+import { findBidsByUserId } from "src/services/bidService";
+
+import { gavel } from "src/assets/icons";
+import { BUTTON_LABELS } from "src/constants";
+
+import "./style.scss";
+
+const BidsTab = () => {
+  const [bids, setBids] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  const { userId } = useUser(); 
+
+  const fetchBids = () => {
+    setLoading(true);
+
+    findBidsByUserId(userId)
+      .then((bids) => {
+        setBids(bids);
+      })
+      .catch((error) => {
+        setError(error.message);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  }
+
+  useEffect(() => {
+    fetchBids();
+  }, [userId]);
+
+  const noItemsComponent = (
```

Comment:
No need to add component in name, EmptyTab is just fine, also I'm not 100% sure, but do we have more cases where this looks almost the same (just different icon and text), if we does, can we create component for that empty state.